woocommerce/woocommerce-android
# Deprecate ic_info_outline_24dp with grey fill color.

Closes: #6863 


### Description

See description in #6863 .

In this PR I've applied the new, `tintable` icon to the places I knew that won't cause any issues as they have applied a tint. For other places - I'm not sure and I can't spend time on this right now.

We also might have a similar issue with other icons but I didn't do research.

### Testing instructions


I recommend opening `Pull requests` feature in Android Studio/Intellij and smoke-check views that have `tintable_ic_info_outline_24dp` applied (it's the new one). 

‼️ You might want to bump AGP version to 7.2.1 to have previews generated in AS (see p1655463089920269-slack-C030U03RC8Y )

### Images/gif


Some samples:

| Before | After |
| --- | --- |
| <img width="188" alt="Screenshot 2022-07-05 at 13 11 05" src="https://user-images.githubusercontent.com/5845095/177318446-24545189-1c75-42a2-a206-51ba503f9e0e.png"> | <img width="126" alt="Screenshot 2022-07-05 at 13 11 15" src="https://user-images.githubusercontent.com/5845095/177318469-d386004a-cca8-41ca-ac16-5a91f5b0ef24.png"> |
| <img width="144" alt="Screenshot 2022-07-05 at 13 12 56" src="https://user-images.githubusercontent.com/5845095/177318503-f44c551e-9bb4-44f5-b69d-d9a08678e1f6.png"> | <img width="147" alt="Screenshot 2022-07-05 at 13 13 15" src="https://user-images.githubusercontent.com/5845095/177318528-ff7b318a-987d-4f48-b469-29ca5f53cd25.png"> |
